tools/text_processing/split_file_on_column/split_file_on_column.xml

```diff
@@ -6,7 +6,8 @@
     <command>
 <![CDATA[
         mkdir tmp_out &&
-        awk -F'\t' '{print > "tmp_out/"\$$column".$infile.ext" }' '$infile'
+
+        awk -F '\t' 'NR==1{hdr=$0;next}f!="tmp_out/"\$$column".$infile.ext"{if(f) close(f); f="tmp_out/"\$$column".$infile.ext";print hdr>f} {print >> f}' '$infile'
```

Comment:
can you add this as an option?

I could imaging a boolean asking the user to prevent the header line (or even the number of header lines)